_data/Atulin/Atulin.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Łukasz Kondracki
+institution: Wrocławska Wyższa Szkoła Informatyki Stosowanej "Horyzont"
```

Comment:
Remove double quotes from "Horyzont".

Reply:
Swapped double-quotes for apostrophes, is that fine?